src/System.Console/tests/RedirectedStream.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Xunit;
+
+public class RedirectedStream : RemoteExecutorTestBase
+{
+    [Fact] // the CI system redirects stdout, so we can only really test the redirected behavior.
```

Comment:
Really?  I don't know how that would even happen... what was it getting piped to?